Composer/packages/client/src/components/Page.tsx

```diff
@@ -81,7 +81,7 @@ export const main = (hasRenderHeaderContent) => css`
 
 export const mainWithGetStarted = css`
   margin-left: 2px;
-  height: calc(100vh - 385px);
+  height: calc(100vh - 517px);
```

Comment:
this seems random, can you extract it to its own variable with a descriptive name?

Reply:
ok!